portfolio/src/main/webapp/script.js

```diff
@@ -27,35 +27,26 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+//Generates a number between 1 and 200
+function randomNumber() {
+    return Math.floor((Math.random()*200)+1);
+}
+
 /*
- In order to access the secret page you have to solve a randomly generated equation.
- Upon the input of a correct answer a button will appear, linking the user to the page.
+ * In order to access the secret page you have to solve a randomly generated equation.
+ * Upon the input of a correct answer a button will appear, linking the user to the page.
  */
 function secretPage(){
   //Generates a number between 1 and 200 for both numbers.
-  this.a = Math.floor((Math.random()*200)+100);
-  this.b = Math.floor((Math.random()*200)+100);
+  var a = randomNumber();
+  var b = randomNumber();
 
 
   //If b is a greater number we flip the values
-  if(this.b > this.a){
-    var hold =this.b;
-    this.b = this.a;
-    this.a = hold;
-  }
-
-  // This section of code includes the possible equations to solve
-  this.div = function(){
-    return this.a / this.b;
-  }
-  this.sub = function(){
-    return this.a - this.b;
-  }
-  this.add = function(){
-    return this.a + this.b;
-  } 
-  this.mul = function(){
-    return this.a * this.b;   
+  if(b > a){
```

Comment:
Try to keep styling consistent
```suggestion
  if(b > a) {
```